src/main/java/org/motechproject/newebodac/service/VaccineeService.java

```diff
@@ -0,0 +1,31 @@
+package org.motechproject.newebodac.service;
+
+import java.util.List;
+import java.util.UUID;
+import org.motechproject.newebodac.dto.VaccineeDto;
+import org.motechproject.newebodac.mapper.VaccineeMapper;
+import org.motechproject.newebodac.repository.VaccineeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class VaccineeService {
+
+  @Autowired
+  private VaccineeRepository vaccineeRepository;
+
+  private static final VaccineeMapper VACCINEE_MAPPER = VaccineeMapper.INSTANCE;
+
+  public VaccineeDto createVaccinee(VaccineeDto vaccinee) {
+    return VACCINEE_MAPPER.toDto(
+        vaccineeRepository.save(VACCINEE_MAPPER.fromDto(vaccinee)));
```

Comment:
you can do it like this in all services, so there will be no need for the javadoc